velox/functions/FunctionRegistry.cpp

```diff
@@ -122,4 +122,23 @@ resolveVectorFunctionWithMetadata(
   return exec::resolveVectorFunctionWithMetadata(functionName, argTypes);
 }
 
+exec::VectorFunctionMetadata getFunctionMetadata(
```

Comment:
getFunctionMetadata and getVectorFunctionMetadata should have consistent signatures. getVectorFunctionMetadata is returning an optional.

Reply:
Updated, thanks.